app/src/main/java/org/oppia/app/databinding/StateAssemblerMarginBindingAdapters.java

```diff
@@ -0,0 +1,107 @@
+package org.oppia.app.databinding;
+
+import android.view.View;
+import android.view.ViewGroup.MarginLayoutParams;
+import androidx.databinding.BindingAdapter;
+
+public class StateAssemblerMarginBindingAdapters {
+  /** Used to set a margin for exploration split-view. */
+  @BindingAdapter( value = {
+    "explorationSplitViewMarginApplicable",
+    "explorationSplitViewMarginStart",
+    "explorationSplitViewMarginTop",
+    "explorationSplitViewMarginEnd",
+    "explorationSplitViewMarginBottom"},
+    requireAll = false
+  )
+  public static void  setExplorationSplitViewMargin(
+    View view,
+    Boolean isApplicable,
+    float marginStart,
+    float marginTop,
+    float marginEnd,
+    float marginBottom
+  ) {
+    setMarginIfApplicable(view, isApplicable, marginStart, marginTop, marginEnd, marginBottom);
+  }
+
+  /** Used to set a margin for exploration view. */
+  @BindingAdapter( value = {
+    "explorationViewMarginApplicable",
+    "explorationViewMarginStart",
+    "explorationViewMarginTop",
+    "explorationViewMarginEnd",
+    "explorationViewMarginBottom"},
```

Comment:
Please fix formatting here (brace should go on the next line & spacing probably will need to be fixed). I think 'value' is also on the wrong line.

Ditto elsewhere.

Reply:
Done :)